docs/user-manual/api/app-download.md

```diff
@@ -20,20 +22,20 @@ curl -H "Authorization: Bearer {accessToken}" -H "Content-Type: application/json
-| `project_id`            | `number`   | ✔️      | The id of the project.                                                                                                                                                |
-| `name`                  | `string`   | ✔️      | The name of the app. Must be less than 1000 characters.                                                                                                               |
-| `scenes`                | `number[]` | ✔️      | A list of scene ids to be included in the app. When you specify scenes then the first scene in the list will be used as the initial scene of the application.         |
-| `branch_id`             | `string`   |          | The id of the branch. If no id is specified the main branch will be used.                                                                                             |
-| `description`           | `string`   |          | The description of the app. Must be less than 10,000 characters.                                                                                                      |
-| `version`               | `string`   |          | The version of the app. Can be a string up to 20 characters.                                                                                                          |
-| `release_notes`         | `string`   |          | Release notes for the app. Can be a string up to 10,000 characters.                                                                                                   |
-| `scripts_concatenate`   | `boolean`  |          | Set it to true if you want scripts to be concatenated.                                                                                                                |
-| `scripts_minify`        | `boolean`  |          | Set it to true if you want scripts to be minified. Defaults to true.                                                                                                  |
-| `scripts_sourcemaps`    | `boolean`  |          | Set it to true if you want script sourcemaps to be generated. Defaults to false.                                                                                      |
-| `optimize_scene_format` | `boolean`  |          | Set it to true if you want scenes to be in an optimized format (see [Optimize Scene Format](/user-manual/optimization/optimizing-scene-format) for more information). |
-| `engine_version`        | `string`   |          | Set it to a Engine version string ([full list of releases](https://github.com/playcanvas/engine/releases)) if a specific version is needed for the app.               |
+| Name                    | Type       | Required | Default   | Description                                                                                                                                                           |
```

Comment:
If you're going to add a new 'Default' column, I think we'll need to update _all_ end point parameter tables.

Reply:
Yeah I was thinking that. Would you like me to propose that as part of this PR?

Reply:
Yeah, that'd be good.

Reply:
PR updated.